pkg/kubelet/dockershim/helpers_linux.go

```diff
@@ -103,12 +104,17 @@ func (ds *dockerService) updateCreateConfig(
 		// TODO: Can we assume the defaults are sane?
 		rOpts := lc.GetResources()
 		if rOpts != nil {
+			max_pids := config.Linux.MaxPids
```

Comment:
no underscores in go variable names

Reply:
Ack. still getting used to this stuff :) thanks! will fix.